apps/prairielearn/src/lib/server-jobs.ts

```diff
@@ -246,3 +247,18 @@ export async function createServerJob(options: CreateServerJobOptions): Promise<
   serverJobs.liveJobs[job_id] = serverJob;
   return serverJob;
 }
+
+const JobOutputSchema = z.object({
+  id: IdSchema,
+  output: z.string().nullable(),
+  status: z.enum(['Running', 'Success', 'Error']),
+});
+type JobOutput = z.infer<typeof JobOutputSchema>;
+
+export async function getJobOutput(jobSequenceId: string): Promise<JobOutput[]> {
+  return await queryRows(
+    sql.select_job_output,
+    { job_sequence_id: jobSequenceId },
+    JobOutputSchema,
```

Comment:
Same here, I'd prefer to have this return a `JobSequenceSchema`.

Reply:
Done in e746ff8.